README.md

```diff
@@ -86,6 +85,14 @@ NOTE: the `license` entry will be `undefined` if we do not yet have license data
 */
 ```
 
+You can get the SVG string for an icon like so:
+```javascript
+import { siSimpleicons } from 'simple-icons/icons';
+import { getSvg } from 'simple-icons/utils';
```

Comment:
I'm not sure about the name, `getSvg` seems to be very general to me. Maybe `getSvgFromSimpleIcon`?

Reply:
The types and function param name should help but that's fine for me if you still prefer it